pkg/utils/mountinfo/mountpoint.go

```diff
@@ -127,3 +136,25 @@ func getBrokenBindMounts(globalMountByName map[string]*Mount, bindMountByName ma
 	}
 	return
 }
+
+type MountPoints []MountPoint
+
+func (mp MountPoints) Len() int { return len(mp) }
+func (mp MountPoints) Less(i, j int) bool {
+	if strings.Contains(mp[i].MountPath, "main") && !strings.Contains(mp[j].MountPath, "main") {
```

Comment:
Use constants for frequently used or significant strings like "main" in the code is a good practice.